doc/api/events.md

```diff
@@ -935,13 +935,415 @@ if the `EventEmitter` emits `'error'`. It removes all listeners when
 exiting the loop. The `value` returned by each iteration is an array
 composed of the emitted event arguments.
 
+## `EventTarget` and `Event` API
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+The global `EventTarget` and `Event` objects are a Node.js-specific adaptation
```

Comment:
Hey, so any reason we'd want to _expose_ EventTarget from Node rather than just implement it internally?

We can expose EventTarget related stuff without supporting the EventTarget constructor or ways to create it which would greatly simplify the maintainer surface for EventTarget?

Reply:
entirely possible if we don't want to allow users to create `EventTarget` instances. It would just be a matter of not exporting the global

Reply:
@jasnell I think that would make this (and the AbortController) stuff a lot easier to land and I'd be very comfortable with an existing (but not exposed) EventTarget in core.

Since our EventTarget provides no way to implement getParent (which I think is good) and other spec required things - I think it's better to not export it (other than on AbortController).

Reply:
Ok, updated to not expose the `Event` and `EventTarget` constructors.